odl/tomo/backends/scikit_radon.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Apr 20 10:48:55 2016
+
+@author: jonasadler
+"""
+
+from odl.discr import uniform_discr_frompartition, uniform_partition
+import numpy as np
+from skimage.transform import radon, iradon
+
+__all__ = ('scikit_radon_forward', 'scikit_radon_back_projector')
+
+def make_scikit_theta(geometry):
+    """ Calculate angles in degrees with ODL scikit conventions. """
```

Comment:
Remove the extra spaces. Also, `scikit_theta` without `make_` would be a better name.